### PR TITLE
Re-push of the numpy transpose entry

### LINKED ISSUE #5369: [Edit] Python:NumPy Ndarray - `.transpose()`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `.transpose()` term entry under NumPy. The entry lives in `content/numpy/concepts/ndarray/terms/transpose/transpose.md`

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) guide when working on the PR for this issue.

Please:

1. Add a ## Codebyte Example section that provides a codebyte example demonstrating the concept in use
2. Ensure that the codebyte example is different than all the other examples in the entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to re-push the numpy transpose entry to resolve conflicts

### Issue Solved

Closes #5369 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [/] All writings are my own.
- [/] My entry follows the Codecademy Docs style guide.
- [/] My changes generate no new warnings.
- [/] I have performed a self-review of my own writing and code.
- [/] I have checked my entry and corrected any misspellings.
- [/] I have made corresponding changes to the documentation if needed.
- [/] I have confirmed my changes are not being pushed from my forked `main` branch.
- [/] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [/] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by arisdelacruz** [comment]
@mamtawardhani @cigar-galaxy82 

Guys, I'm not sure if I did it right. It appears that the rebase problems was replaced by the contains issue

Please provide further guidance. Thanks

File: content/numpy/concepts/ndarray/terms/transpose/transpose.md [hunk 1]
```diff
@@ -52,3 +52,13 @@ This produces the following output:
  [2 5]
  [3 6]]
 ```
+
```
Comment (Radhika-okhade) [inline content/numpy/concepts/ndarray/terms/transpose/transpose.md:55]: Please provide a description of what the below example does.

**Comment by Radhika-okhade** [comment]
Hey! @arisdelacruz I have added one comment could you please go through it?

File: content/numpy/concepts/ndarray/terms/transpose/transpose.md (hunk 1 above)
Comment (arisdelacruz) [reply content/numpy/concepts/ndarray/terms/transpose/transpose.md:55]: @Radhika-okhade done

thanks

**Comment by arisdelacruz** [comment]
> Hey! @arisdelacruz I have added one comment could you please go through it?

done. Thanks much

**Review by Radhika-okhade — APPROVED** [review]

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @arisdelacruz, seems like you have added some changes to another file by mistake. Please remove the other file changes. Let me know if you need any help in the process.

File: content/numpy/concepts/ndarray/terms/transpose/transpose.md [hunk 2]
```diff
@@ -52,3 +52,17 @@ This produces the following output:
...
+This codebyte example demonstrates how to create and manipulate a multi-dimensional array in Python using NumPy’s `transpose` function.
+
+```codebyte/python
+import numpy as np
+
+a = np.ones((2, 3, 4, 5))
+np.transpose(a).shape
+(5, 4, 3, 2)
+```
```
Comment (PragatiVerma18) [inline content/numpy/concepts/ndarray/terms/transpose/transpose.md:68]: The output in the last line is not generated by the code itself, so we will need to print the output to verify it directly:

```suggestion
```codebyte/python
import numpy as np

a = np.ones((2, 3, 4, 5))
print(np.transpose(a).shape)
```
```

File: content/numpy/concepts/ndarray/terms/transpose/transpose.md [hunk 3]
```diff
@@ -52,3 +52,17 @@ This produces the following output:
  [2 5]
  [3 6]]
 ```
+
+In the above output, the original array is a 2x3 matrix, and the transposed array is a 3x2 matrix.
+
+## Codebyte example
+
+This codebyte example demonstrates how to create and manipulate a multi-dimensional array in Python using NumPy’s `transpose` function.
```
Comment (PragatiVerma18) [inline content/numpy/concepts/ndarray/terms/transpose/transpose.md:60]: We should mention the exact function name alongwith `.` and parantheses -

```suggestion
This codebyte example demonstrates how to create and manipulate a multi-dimensional array in Python using NumPy’s `.transpose()` function.
```

**Comment by arisdelacruz** [comment]
> Hey @arisdelacruz, seems like you have added some changes to another file by mistake. Please remove the other file changes. Let me know if you need any help in the process.

@PragatiVerma18 

It appears that I cannot remove the contains entry for swift. Shall I create a fresh pull? Thanks

**Review by PragatiVerma18 — APPROVED** [review]
Hey @arisdelacruz, I have restored the swift file changes. The PR is now good to go!

**Pull request merged**